opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -104,10 +145,15 @@ def __init__(
         metric_readers: Sequence[MetricReader] = (),
         resource: Resource = Resource.create({}),
         shutdown_on_exit: bool = True,
+        measurement_consumer_class: MeasurementConsumer = (
```

Comment:
I don't think we should expose this to the user for now, can you remove?

Reply:
Ok, but I think that we will need a mechanism to do that. Opening a corresponding issue.